cycle_docker_compose.sh

```diff
@@ -3,6 +3,9 @@
 echo "Starting docker-compose, press Ctrl+D to cycle docker compose using a down-up cycle"
 echo "Press Ctrl+C (once) to stop"
 sleep 1
+docker build -t algorithm_io app/tests/resources/gc_demo_algorithm/
+docker save algorithm_io -o app/tests/resources/gc_demo_algorithm/algorithm_io.tar
+chmod a+r app/tests/resources/gc_demo_algorithm/algorithm_io.tar
```

Comment:
Could this be moved to the makefile?

`app/tests/resources/gc_demo_algorithm/algorithm_io.tar` needs to be added to gitignore.

The tar file original stored has been squashed out, right?

Reply:
Also, do we really need it on every development startup?

Reply:
That's true, we could just make it a separate Make target that you'd have to specifically run if you want the actual container to be there.